packages/taquito/test/contract/big-map.spec.ts

```diff
@@ -7,169 +7,176 @@ import { BigMapAbstraction } from '../../src/contract/big-map';
  * BigMapAbstraction test
  */
 describe('BigMapAbstraction test', () => {
-    let rpcContractProvider: {
-        getBigMapKeysByID: jest.Mock<any, any>;
-        getStorage: jest.Mock<any, any>;
-        getBigMapKey: jest.Mock<any, any>;
-        getBigMapKeyByID: jest.Mock<any, any>;
+  let rpcContractProvider: {
+    getBigMapKeysByID: jest.Mock<any, any>;
```

Comment:
are these just lint changes?

Reply:
yes I only manually changed line1 from `'@taquito/http-utils';` to `'@taquito/http-core';`